app/gosqs/send_message_test.go

```diff
@@ -0,0 +1,111 @@
+package gosqs
+
+import (
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/Admiral-Piett/goaws/app"
+	"github.com/Admiral-Piett/goaws/app/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSendMessage_MaximumMessageSize_Success(t *testing.T) {
+	req, err := http.NewRequest("POST", "/", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	app.SyncQueues.Queues["test_max_message_size"] =
+		&app.Queue{Name: "test_max_message_size", MaximumMessageSize: 100}
+
+	form := url.Values{}
+	form.Add("Action", "SendMessage")
+	form.Add("QueueUrl", "http://localhost:4100/queue/test_max_message_size")
+	form.Add("MessageBody", "test%20message%20body%201")
+	form.Add("Version", "2012-11-05")
+	req.PostForm = form
+
+	status, response := SendMessageV1(req)
```

Comment:
Checkout how I did these kinds of tests.  Let's get into that pattern if we can.  I intentionally made `utils. REQUEST_TRANSFORMER` a global variable so we could swap it in tests at run time.  Just so every unit test didn't pass through that each time.  

I guess it's fine if they do, but this request handling stuff, I want to get it out of the handler logic as much as we can - still chewing on how to do it ultimately, but for now, I prefer this pattern below?

https://github.com/Admiral-Piett/goaws/blob/feature-aws-json/app/gosqs/create_queue_test.go#L30-L34

Reply:
I updated the UTs!
https://github.com/Admiral-Piett/goaws/pull/293#discussion_r1601627025